backend/app/src/main/java/com/onair/hearit/app/playinghistory/dto/PlayingHistoryRequest.java

```diff
@@ -13,5 +14,8 @@ public record PlayingHistoryRequest(
         if (lastPlayTime == null || lastPlayTime < 0) {
             throw new InvalidInputException("마지막 재생 기록은 0 이상이어야 합니다.");
         }
+        if (clientEventTime != null && clientEventTime < 0) {
+            throw new InvalidInputException("요청 시간은 0 이상이어야 합니다.");
+        }
```

Comment:
hearitId랑 lastPlayTime은 null이면 안되는거니까 validation으로 넣고, 
client eventTime만 지금 로직으로 두는거 어떤가요? 
그러면 clientEventTime만 nullable인게 더 잘 보일 것 같아요. 글고 주석으로 하위호환성때문에 nullable 허용해주는거 주석으로 쓰면 좀 더 좋을듯염 이건 다른 사람들의 이야기로 들어보면 조을듯

```suggestion
public record PlayingHistoryRequest(
        @NotNull Long hearitId,
        @NotNull Long lastPlayTime,
        Long clientEventTime
) {
    public PlayingHistoryRequest {
        if (lastPlayTime < 0) {
            throw new InvalidInputException("마지막 재생 기록은 0 이상이어야 합니다.");
        }
        if (clientEventTime != null && clientEventTime < 0) {
            throw new InvalidInputException("요청 시간은 0 이상이어야 합니다.");
        }
```

Reply:
좋은거같음요

Reply:
훨씬 가독성 좋네용. 반영해두고 주석도 추가해두었습니다!